src/ensembl/src/content/app/browser/browserHelper.ts

```diff
@@ -24,6 +24,19 @@ export function getChrLocationStr(
   return `${chrCode}:${startBp}-${endBp}`;
 }
 
+export type ErrorMessages = {
+  genomeIdError: string | undefined;
+  regionParamError: string | undefined;
+  regionError: string | undefined;
+  startError: string | undefined;
+  endError: string | undefined;
```

Comment:
Here and below, would it be simpler to use `endError?: string` instead of specifying `undefined`?

Reply:
The types of these error properties have now changed to `string | null`. So all of them will exist in the returned `RegionValidationMessages` object.